upgrade_guide/README.md

```diff
@@ -9,6 +9,26 @@ request cannot be executed because all nodes tried were busy. Previously you wou
 `NoNodeAvailableException` but you will now get back an `AllNodesFailedException` where the
 `getAllErrors` map contains a `NodeUnavailableException` for that node.
 
+#### Esri Geometry dependency now optional
```

Comment:
Now that we are doing this for Esri, I wonder why we wouldn't do the same for Jackson and Reactive Streams. Indeed, these 2 dependencies are handled the same way as Esri used to be handled: declared as mandatory in pom.xml, but made optional by the magic of reflection. 

Do you mind opening a ticket for that, if it doesn't exist yet?

Reply:
See above re: my hesitation about doing so with Jackson but the reactive streams dependency would seem to make a lot of sense on this front.  I'll file a ticket for that.

Reply:
Ticket is https://datastax-oss.atlassian.net/browse/JAVA-2981